rasa_core/channels/facebook.py

```diff
@@ -50,6 +57,9 @@ def message(self, message: Dict[Text, Any]) -> None:
         elif self._is_audio_message(message):
```

Comment:
Since your `is_attachment_message` covers the `is_audio_message`, we can remove `_is_audio_message` and only use your implementation.

Reply:
I'm not sure it's safe to do that. Maybe we should state that this function is deprecated.

Reply:
I checked and it's the same behavior: it adds the url of the attachment as text. So you can safely drop the audio part.